.gitignore

```diff
@@ -33,6 +33,4 @@ build/
 ### VS Code ###
 .vscode/
 
-.mvn
```

Comment:
are you taking this off on purpose?

Reply:
Yes so we can enable the maven git versioning extension